packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFUIDelegateHostApi.m

```diff
@@ -61,7 +61,7 @@ - (void)requestMediaCapturePermissionForDelegateWithIdentifier:(FWFUIDelegate *)
                                                        webView:(WKWebView *)webView
                                                         origin:(WKSecurityOrigin *)origin
                                                          frame:(WKFrameInfo *)frame
-                                                          type:(FWFWKMediaCaptureType)type
+                                                          type:(WKMediaCaptureType)type
```

Comment:
This was not causing an crash since all enums are represented by an `NSInteger`. However this was causing the lint error:

```
error: implicit conversion from enumeration type 'FWFWKMediaCaptureType' (aka 'enum FWFWKMediaCaptureType') to different enumeration type 'WKMediaCaptureType' (aka 'enum WKMediaCaptureType') [-Werror,-Wenum-conversion]
````